scripts/cleanup_failed_registrations.py

```diff
@@ -31,6 +31,8 @@ def remove_failed_registrations(dry_run=True):
     if not dry_run:
         for f in failed:
             logging.info('Cleaning {}'.format(f))
+            if not f.registered_from:
```

Comment:
It shouldn't happen, but these archive jobs shouldn't be getting stuck.  

We are setting node.registered_from = None when deleting failed registrations, so this is a potential (albeit broken) state we need to handle.

Will add logging